docs/src/pages/components/slider/DiscreteSlider.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/lab/Slider';
+
+const useStyles = makeStyles({
+  root: {
+    width: 300,
+  },
+});
+
+const marks = [
+  {
+    value: 0,
+    label: '0°C',
+  },
+  {
+    value: 20,
+    label: '20°C',
+  },
+  {
+    value: 37,
```

Comment:
You can never get to 37°C since the step is 10

Reply:
I don't know what the default behavior should be here. I definitely think that there is room for both. What do you think? I'm happy to start by supporting only one case and wait for somebody issue.

Reply:
I'm fine with keeping it as is, just wanted to flag it up :)

Reply:
I think that we can keep the component in the lab for a month, so we collect more feedback from people. I'm doing too many changes in the pull request to get everything right. It's 100% guarentee follow up changes will be needed. One step at the time.